path_config.go

```diff
@@ -0,0 +1,198 @@
+package jwtauth
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"errors"
+	"net/http"
+
+	"context"
+
+	oidc "github.com/coreos/go-oidc"
+	"github.com/hashicorp/errwrap"
+	cleanhttp "github.com/hashicorp/go-cleanhttp"
+	"github.com/hashicorp/vault/helper/certutil"
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+	"golang.org/x/oauth2"
+)
+
+func pathConfig(b *jwtAuthBackend) *framework.Path {
+	return &framework.Path{
+		Pattern: `config`,
+		Fields: map[string]*framework.FieldSchema{
+			"oidc_issuer_url": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "OIDC issuer URL, without any .well-known component (base path)",
```

Comment:
Might be worth adding a note here and to the pubkeys description below that only one of them could be set?

Reply:
Done